test/catalog/constraints_test.cpp

```diff
@@ -26,12 +30,30 @@
 #include "storage/table_factory.h"
 #include "storage/database.h"
 #include "index/index_factory.h"
-
 #include "catalog/constraints_tests_util.h"
 
+#include "common/harness.h"
+#include "common/macros.h"
+#include "common/logger.h"
+#include "optimizer/simple_optimizer.h"
+#include "executor/logical_tile_factory.h"
+#include "executor/plan_executor.h"
+#include "executor/seq_scan_executor.h"
+#include "executor/update_executor.h"
+#include "planner/plan_util.h"
+#include "parser/postgresparser.h"
+
 #define NOTNULL_TEST
```

Comment:
What are these different `#defines` for? Why wouldn't you want to just run all of the tests?